package.json

```diff
@@ -46,7 +46,7 @@
     "style-loader": "^1.1.3",
     "terser-webpack-plugin": "^1.1.0",
     "tmp": "^0.0.33",
-    "webpack": "^4.20.0",
+    "webpack": "^4.22.0",
```

Comment:
Fixes a change in how some `0.js` and `1.js` split files are rendered - a fix was introduced in 4.22.0.